release-notes-3.4.0.md

```diff
@@ -1,8 +1,7 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
-
+- [SCMOD-8955]: Updated worker framework version
```

Comment:
Include the version the worker framework has been updated  to:

 Worker Framework updated to [x.x.x.](https://github.com/WorkerFramework/worker-framework/releases/tag/vx.x.x)

Reply:
Done